services/data_plane_manager/src/test/java/com/futurewei/alcor/dataplane/config/TestConfig.java

```diff
@@ -58,4 +58,6 @@
     public static String availabilityZone = "az1";
     public static String primaryDns = "1.1.1.1";
     public static String secondaryDns = "2.2.2.2";
+    public static String deviceId = "device-id-0";
+    public static String deviceOwner = "device-owner-0";
```

Comment:
@VanderChen This change is also show in another PR.

Reply:
These are not my changes, I am not sure how they are showing up my changes. Let me see what might have gone wrong.